src/prefect/tasks/great_expectations/checkpoints.py

```diff
@@ -97,7 +105,9 @@ class RunGreatExpectationsValidation(Task):
     def __init__(
         self,
         checkpoint_name: str = None,
-        context: "ge.DataContext" = None,
+        ge_checkpoint: Checkpoint = None,
+        checkpoint_kwargs: Dict = {},
```

Comment:
If you're typing a dict for kwargs you can indicate that the keys must be strings.

```suggestion
        checkpoint_kwargs: Dict[str, Any] = None,
```


You should also avoid having a dict as a default as it can cause issues later. You can set the default at runtime/init
```python
checkpoint_kwargs = checkpoint_kwargs or {}
```